src/samples/Authorization/ManualSignIn/Program.cs

```diff
@@ -36,16 +39,28 @@
 
 var app = builder.Build();
 
+// Configure the HTTP request pipeline.
+app.UseRouting();
+app.MapPost("/api/messages", async (HttpRequest request, HttpResponse response, IAgentHttpAdapter adapter, IAgent agent, CancellationToken cancellationToken) =>
+    {
+        await adapter.ProcessAsync(request, response, agent, cancellationToken);
+    })
+    .AllowAnonymous();
+
+// Setup development host and allow use of a local launchSettings.json file
 if (app.Environment.IsDevelopment())
 {
-    app.MapGet("/", () => "Microsoft Agents SDK Sample");
-    app.UseDeveloperExceptionPage();
-    app.MapControllers().AllowAnonymous();
+    string launchSettingsPath = Path.Combine(app.Environment.ContentRootPath, "Properties", "launchSettings.json");
+    if (!File.Exists(launchSettingsPath))
```

Comment:
mmhh.. this is weird.. I dont think it's used.

If we want to discuss how to configure the port, we should do it in a different PR, this one is for Auth and should use whatever we are using in the other samples.

Reply:
Ack.. set it for brevity for now.